source/manual/republishing-content.html.md

```diff
@@ -15,15 +15,19 @@ This process varies per app and requires
 
 ## Whitehall
 
-If the document is in Whitehall, there is a Rake task you can run:
+If the documents are in Whitehall, there are Rake tasks you can run as outlined below. Try to pick the one most focused to the scope of what you need to republish to avoid unnecessary load. You can monitor the effect on the publishing queue via these Grafana dashboards:
+
+- [integration][https://grafana.integration.publishing.service.gov.uk/dashboard/file/sidekiq.json?refresh=1m&orgId=1&var-Application=whitehall&var-Interval=$__auto_interval]
```

Comment:
these are not rendering as links, possibly because of the bullet points?

Reply:
Fixed - it needed () instead of [] on the second half of the link markup as it's not following the pattern of the others where the link is defined below the body.